src/services/llms/base.py

```diff
@@ -58,22 +58,26 @@ def _clean_response_text(self, text: str) -> str:
 
     def _parse_response(self, response_text: str) -> List[Dict[str, str]]:
         """
-        Parse the cleaned response text into structured data.
+        Parse the cleaned response text into structured review comments.
         
         Args:
-            response_text: Cleaned response text
+            response_text: JSON formatted response text from LLM
             
         Returns:
-            List[Dict[str, str]]: Parsed review comments
```

Comment:
Mô tả giá trị trả về cũng nên tổng quát hơn. Thay vì `List of valid review comments containing lineNumber and reviewComment`, hãy dùng `List of dictionaries representing the parsed data`.